### PR TITLE
Fix `skipping test that requires root` issue

### PR DESCRIPTION
**Pull request by chenrui333**
Relates to https://github.com/containerd/zfs/pull/28/commits/220e87a275c0e355562328c72a294bf84f5350b3

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ before_script:
 script:
     - DCO_VERBOSITY=-q ../project/script/validate/dco
     - ../project/script/validate/fileheader ../project/
-    - go test -v -race -covermode=atomic -coverprofile=coverage.txt ./...
+    - sudo go test -v -race -covermode=atomic -coverprofile=coverage.txt ./...
```
Comment (fuweid) [inline .travis.yml]: `sudo PATH=$PATH GOPATH=$GOPATH` will help

**Comment by AkihiroSuda** [comment]
ping @chenrui333

**Comment by chenrui333** [comment]
Thanks the reminder, picking it up now.

**Comment by AkihiroSuda** [comment]
CI still failing. maybe `go test -exec sudo` works.

**Comment by AkihiroSuda** [comment]
Also, don't drop `-test.root -test.v`

**Comment by chenrui333** [comment]
I am going to spin up a new repo and test it separately.
https://travis-ci.com/chenrui333/aufs-test

**Comment by chenrui333** [comment]
Looks like travis does not have debug mode by default for public repo, I have to switch to company private repo to debug the build.

**Comment by chenrui333** [comment]
@fuweid @AkihiroSuda should be good now, thanks for the help!

File: .travis.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ before_script:
 script:
     - DCO_VERBOSITY=-q ../project/script/validate/dco
     - ../project/script/validate/fileheader ../project/
-    - go test -v -race -covermode=atomic -coverprofile=coverage.txt ./...
+    - sudo $GOROOT/bin/go test -test.root -test.v -race -covermode=atomic -coverprofile=coverage.txt ./...
```
Comment (chenrui333) [inline .travis.yml:17]: Tried to use `export PATH=$GOROOT/bin:$PATH`, seems not quite work out. Let me know if you guys prefer to have a comment for this line as well.

Comment (chenrui333) [reply .travis.yml:17]:
```
travis@travis-job-f1c1eeb9-d213-4c5b-bd66-aa15232af65a:~/gopath/src/github.com/meetup/aufs-test$ export PATH=$GOROOT/bin:$PATH
travis@travis-job-f1c1eeb9-d213-4c5b-bd66-aa15232af65a:~/gopath/src/github.com/meetup/aufs-test$ sudo go test -test.root -test.v -race -covermode=atomic -coverprofile=coverage.txt ./...
sudo: go: command not found
```

Comment (AkihiroSuda) [reply .travis.yml:17]: Why not simply use `go test -c` as in former revisions?

Comment (chenrui333) [reply .travis.yml:17]: I thought we have `sudo: go: command not found` issue?

Comment (AkihiroSuda) [reply .travis.yml:17]: So previous version didn't have that issue because we don't need `sudo go` for `go test -c`

Comment (AkihiroSuda) [reply .travis.yml:17]: ping

Comment (AkihiroSuda) [reply .travis.yml:17]: png @chenrui333

**Comment by AkihiroSuda** [comment]
ping @chenrui333

**Comment by AkihiroSuda** [comment]
We are switching away from Travis (https://github.com/containerd/aufs/pull/25), so closing this

**Pull request closed without merging**